packages/webcert/src/feature/certificate/Buttons/RevokeCertificateButton.tsx

```diff
@@ -54,7 +58,7 @@ const RevokeCertificateButton: React.FC<Props> = ({ name, description, enabled,
       modalTitle={metadata?.status === CertificateStatus.LOCKED ? 'Makulera låst utkast' : 'Makulera intyg'}
       onConfirm={handleDispatch}
       confirmButtonText="Makulera">
-      <RevokeCertificateModalContent onChange={handleRevokeForm} type={metadata?.type} />
+      {!isDodsbevis ? <RevokeCertificateModalContent onChange={handleRevokeForm} type={metadata?.type} /> : <RevokeDodsbevisModalContent />}
```

Comment:
Easier to read the code if you remove the negating operand ('!') and just change order to the revoke modals.